content/en/logs/error_tracking/browser_and_mobile.md

```diff
@@ -44,20 +44,11 @@ If you have not setup the Datadog Browser SDK yet, follow the [in-app setup inst
    })
    ```
 
-4. You must add the following snippet near initialization to catch and submit all uncaught exceptions to Datadog:
+4. To log a caught exception yourself, you may optionally use:
 
    ```javascript
-   window.onerror = function (message, source, lineno, colno, error) {
-       datadogLogs.logger.error(error?.message || '', {
-           error: { stack: error?.stack },
-       });
-   };
-   ```
-5. To log a caught exception yourself, you may optionally use:
-
-   ```javascript
-   const e = new Error('an exception occurred');
-   datadogLogs.logger.error(e.message, {'error': {'stack': e.stack}});
+   const err = new Error('an exception occurred');
+   datadogLogs.logger.error("an error occured", {usr: {id: 123}}, err);
```

Comment:
The section's title mentions 'caught exceptions', so I suggest to use an example with a try/catch block.

```suggestion
try {
  ...
  throw new Error('Wrong behavior');
  ...
} catch (ex) {
  datadogLogs.logger.error('Error occurred', { usr: { id: 123}}, ex);
}
```

Reply:
Good call done in https://github.com/DataDog/documentation/pull/17416/commits/83483d0f350257895d79548f650eb48116a02b01